extension/src/experiments/webview/messages.ts

```diff
@@ -267,20 +251,19 @@ export class WebviewMessages {
         this.experiments.getAvailableBranchesToShow().length > 0,
       hasCheckpoints: this.experiments.hasCheckpoints(),
       hasColumns: this.columns.hasNonDefaultColumns(),
-      hasConfig: this.hasConfig,
+      hasConfig: !!(this.pipeline.hasStage() || this.pipeline.hasPipeline()),
```

Comment:
[I] probably want to hide this logic in `pipeline` can also probably be reduced to `hasPipeline`